processing/src/main/java/io/druid/query/aggregation/hyperloglog/HyperUniquesSerde.java

```diff
@@ -91,9 +80,7 @@ public HyperLogLogCollector extractValue(InputRow inputRow, String metricName)
           }
 
           for (String dimensionValue : dimValues) {
-            collector.add(
-                hashFn.hashBytes(StringUtils.toUtf8(dimensionValue)).asBytes()
-            );
+            collector.add(dimensionValue);
```

Comment:
What's the motivation behind removing the hashFn parameter of the serde? The collector is supposed to be hash-function agnostic, which is why the serde applied it before handing over the `hashedValue`.